### DISCUSSION TITLE
Does mockoon-cli support templates?

### DISCUSSION THREAD

**Discussion by PureKrome** [mockoon/mockoon#1447] [General]
SO now that I have mockoon [working via docker compose](https://github.com/mockoon/mockoon/discussions/1410), our data file has some templating in the response:

```
"body": "{\n  \"statusCode\": 200,\n  \"message\": \"OK\",\n  \"data\": {\n    \"isResetPassword\": false,\n    \"userId\": \"{{guid}}\",\n    \"session\": {\n      \"AccessToken\": \"{{data 'RandomToken'}}\",\n      \"ExpiresIn\": 86400,\n      \"TokenType\": \"Bearer\",\n      \"RefreshToken\": \"{{data 'RandomToken'}}\",\n      \"IdToken\": \"{{data 'RandomToken'}}\"\n    }\n  }\n}",
```
specifically:

- `{{guid}}`
- `{{data 'RandomToken'}}`

is this suppose to work?

when i see the response content, it's `null`.

when i replace it with:

```
"body": "hello world"
```
the response content is `"hello world"`

Is this by design or have I messed up, again?

**Comment by 255kb** [comment]
Yes, the CLI supports 100% of the desktop app features. I just tested your body both in the desktop app and the CLI, just in case, and it worked perfectly. 
Maybe there is something going on with your setup? If you could [share the full environment](https://mockoon.com/docs/latest/mockoon-data-files/data-storage-location/) that would be useful. 

Thanks.

**Reply by PureKrome** [reply]
Confirmed that it is working fine - must have been a weird issue with a typo or something. But yes, all working great. Thank you kindly.
